aws/v15.0.0/release.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
   name: v15.0.0
 spec:
   apps:
-  - componentVersion: 1.1.0
+  - componentVersion: 1.1.3
```

Comment:
Why was the `componentVersion` changed for aws-ebs-cni-driver in v15.0.0? In the release diff of v15.1.0 `componentVersion` is set to 1.1.1. It looks to me like a mistake.

Reply:
This is what I saw in the changelog but could not find which is the one really there. @njuettner do you know?

Reply:
In release v15 we introduced aws-ebs-csi-driver, component version should be `1.1.0`

Reply:
I set the version to 1.1.0 for v15.0.0 .